### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jwhonce**
Create contributing guideline file.

Needs more work, but this is a start.

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

**Comment by jwhonce** [comment]
@nick-lehmann FYI

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ 
+## Where to find other contributors
+ - For general questions and discussion, please use the IRC #podman channel on irc.freenode.net.
+ - For discussions around issues/bugs and features, you can use the GitHub [issues](https://github.com/containers/podman-py/issues) and [PRs](https://github.com/containers/podman-py/pulls) tracking system.
+ 
+## Coding conventions
+ - Pass pylint
+   - exceptions are possible but you will need to make a good argument
+ - use spaces not tabs for indentation
+ - This is open source software. Consider the people who will read your code, and make it look nice for them. It's sort of like driving a car: Perhaps you love doing donuts when you're alone, but with passengers the goal is to make the ride as smooth as possible.
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md]: hmmm, donuts!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# How to contribute
+
+Thank you in your interest in this project.  We need your help to make it successful. 
+
+You may also want to look at:
+ - https://github.com/containers/libpod
+ - https://podman.readthedocs.io/en/latest/Reference.html
+ -
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md]: empty bullet her

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# How to contribute
+
+Thank you in your interest in this project.  We need your help to make it successful.
```
Comment (TomSweeneyRedHat) [inline CONTRIBUTING.md]: suggest "this project" -> "the podman-py project"

**Comment by rhatdan** [comment]
LGTM
I am going to merge, but we should probably bring up information about the podman.io mailing lists and freenode: #podman

**Pull request merged**